test/common/scale_up_replicas_apicurito.go

```diff
@@ -46,16 +48,52 @@ func TestReplicasInApicurito(t *testing.T, ctx *TestingContext) {
 		t.Fatalf("Incorrect number of replicas : %v", err)
 	}
 
-	apicuritoCR, err = updateApicurito(ctx.Client, scaleDownApicuritoReplicas, t)
+	t.Log("Checking replicas are ready")
+	if err := checkReplicasAreReady(ctx, t, scaleUpApicuritoReplicas); err != nil {
+		t.Fatalf("Replicas weren't ready within timeout")
+	}
+
+	apicuritoCR, err = updateApicurito(ctx, scaleDownApicuritoReplicas, t)
 	if err != nil {
 		t.Fatalf("Unable to update : %v", err)
 	}
 
+	t.Log("Checking replicas are ready")
+	if err := checkReplicasAreReady(ctx, t, scaleDownApicuritoReplicas); err != nil {
+		t.Fatalf("Replicas weren't ready within timeout")
+	}
+
 	t.Log("Checking correct number of updated replicas are set")
 	if err := checkNumberOfReplicasAgainstValueApicurito(apicuritoCR, ctx, numberOfApicuritoReplicas, retryIntervalApicurito, timeoutApicurito, t); err != nil {
 		t.Fatalf("Incorrect number of replicas : %v", err)
 	}
 
+	t.Log("Checking replicas are ready")
+	if err := checkReplicasAreReady(ctx, t, numberOfApicuritoReplicas); err != nil {
+		t.Fatalf("Replicas weren't ready within timeout")
+	}
+
+}
+
+func checkReplicasAreReady(dynClient *TestingContext, t *testing.T, replicas int32) error {
+	deployment, err := dynClient.KubeClient.AppsV1().Deployments(apicuritoNamespace).Get(apicuritoName, metav1.GetOptions{})
```

Comment:
This should not be required with the same get inside the poll on line 83 https://github.com/integr8ly/integreatly-operator/pull/754/files#diff-c390ece00887543e97f361f7e04d648aR83